packages/html/src/multiselecttree/multiselecttree.spec.tsx

```diff
@@ -55,7 +55,7 @@ export type KendoMultiSelectTreeProps = KendoMultiSelectTreeOptions & {
     dir?: 'ltr' | 'rtl';
     adaptive?: boolean;
     adaptiveSettings?: KendoActionSheetProps;
-    adaptiveHeaderSettings?: KendoActionSheetHeaderProps;
+    adaptiveTitle?: string;
```

Comment:
In my opinion adaptiveTitle is not directly related to the MulstSelectTree component, it relates to the ActionSheet itself and I would suggest to skip this prop